src/pages/Timetable/components/MyLectureTimetable/TotalGrades.tsx

```diff
@@ -0,0 +1,19 @@
+import { LectureInfo, TimetableLectureInfo } from 'interfaces/Lecture';
+import styles from './MyLectureTimetable.module.scss';
+
+function TotalGrades(props: { myLectureList: TimetableLectureInfo[] | LectureInfo[] }) {
+  let total = 0;
+  // eslint-disable-next-line react/destructuring-assignment
+  props.myLectureList.forEach((item) => {
+    total += parseInt(item.grades, 10);
+  });
```

Comment:
```suggestion
 myLectureList.reduce((acc, lecture) => acc + parseInt(lecture.grades, 10), 0);
```
개인적으로 reduce가 더 깔끔하다고 생각해서 `reduce`를 사용하는 것도 괜찮을 방법인 것 같아요!
myLectureList가 변할 때 마다 값이 변하기 때문에 `useMemo`를 사용해볼만 한 것 같아요!

Reply:
반영했습니다! c7aea3eff6b7c42518a0855c5f12503436fd585c